gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -239,6 +240,22 @@ typedef struct
   int rate_d; /**< framerate is in fraction, which is numerator/denominator */
 } GstTensorsConfig;
 
+/**
+ * @brief Internal data structure for sparse tensor info
+ */
+typedef struct
+{
+  uint32_t length; /**< the number of "non-zero" elements */
```

Comment:
Q.  It's ambiguous to me. Is `length` commonly used terminology in sparse tensor or matrix?

Reply:
No, there are no common term I found. `size`, `length`, and `num_elements` were candidates. Maybe other terms be more suitable?

Reply:
Scipy uses `nnz`, FYI

Reply:
Then, let's use `nnz`, `non_zeros`, or other intuitive word.

Reply:
`nnz` seems cool. I chose it. PTAL